app.rb

```diff
@@ -0,0 +1,145 @@
+require_relative 'person'
+require_relative 'rental'
+require_relative 'book'
+require_relative 'student'
+require_relative 'teacher'
+require_relative 'classroom'
+require 'date'
+
+class App
+  attr_reader :books, :people, :rentals
+
+  def initialize
+    @books = []
+    @people = []
+    @rentals = []
+  end
+
+  # list all books
+  def list_books
+    if @books.length.positive?
+      @books.each { |book, index| puts "#{index}. Title: #{book.title}, Author: #{book.author}" }
+    else
+      puts 'No book available yet'
+    end
+  end
+
+  # list all people
+  def list_people
+    if @people.length.positive?
+      @people.each do |person|
+        puts "[#{person.class}] Name: #{person.name}, Age: #{person.age} years, ID: #{person.id}"
+      end
+    else
+      puts 'No person added yet'
+    end
+  end
+
+  # create a person
+  def create_person
+    print 'Do you want to create a student (1) or a teacher (2)? [Input the number]: '
+    person_type = gets.chomp
+
+    if person_type != '1' && person_type != '2'
+      puts 'Invalid option'
```

Comment:
- [ ] Dear @Becky449, You did a great job, But while creating a person whichever option I provide I could not create a person. It throws  ` wrong number of arguments (given 3, expected 1..2) (ArgumentError)`. Please make sure it's possible to create a user before the next review request. 
![image](https://user-images.githubusercontent.com/69077061/232185360-9be4cf30-58e4-4695-84c8-6e6ff1f60219.png)

- [ ] Because of this I couldn't verify the functionality of the remaining tasks, Please also check all the functionalities are working as expected.